app/Models/User.php

```diff
@@ -841,4 +842,25 @@ private function correctPhoneNumber($phone)
         }
         return $phone;
     }
+
+    public function events()
+    {
+        $events = $this->belongsToMany('App\Models\Event', 'UserEvent', 'user_id', 'event_id')
+            ->where('starts_on', '>', date('Y-m-d H:i:s', mktime(1, 1, 1, 1, 1, $this->year())))
```

Comment:
use `->where('starts_on', '>', $this->yearStartTime())`
Year start is not from Jan 1st - but from when the academic year start in mad(generally around july, augest).

Reply:
okay 👍